library/core/tests/iter/traits/marker.rs

```diff
@@ -0,0 +1,17 @@
+#[test]
+fn test_fuse() {
```

Comment:
Why not in `fuse.rs`?

Reply:
My bad, Lol, I think I might have wrote marker in the input file for my splitter program to remind myself to come back to it, but I lost it amidst the chaos.